backends-velox/src/test/scala/org/apache/spark/sql/execution/VeloxParquetWriteForHiveSuite.scala

```diff
@@ -87,6 +87,21 @@ class VeloxParquetWriteForHiveSuite extends GlutenQueryTest with SQLTestUtils {
         _.getMessage.toString.contains("Use Gluten parquet write for hive")) == native)
   }
 
+  test("test hive static partition write table") {
+    withTable("t") {
+      spark.sql(
+        "CREATE TABLE t (c int, d long, e long)" +
+          " STORED AS PARQUET partitioned by (c, d)")
+      withSQLConf("spark.sql.hive.convertMetastoreParquet" -> "true") {
```

Comment:
Should here enable `spark.gluten.sql.native.writer.enabled` to verify the actual native write?

Reply:
@zjuwangg This config have been configured [here](https://github.com/oap-project/gluten/blob/fc79021242a3c3df59ac91040d5e6c69f9968029/backends-velox/src/test/scala/org/apache/spark/sql/execution/VeloxParquetWriteForHiveSuite.scala#L77).